fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -155,6 +161,8 @@
 
     static final String ACCEPT_EXTERNAL_CONTENT = "Accept-External-Content-Handling";
 
+    public static final String ACCEPT_DATETIME = "Accept-Datetime";
```

Comment:
I would suggest moving this constant into ContentExposing Resource and then using it to replace the string at ContentExposingResource:251